modules/vpc-network/main.tf

```diff
@@ -53,7 +53,11 @@ resource "google_compute_subnetwork" "vpc_subnetwork_public" {
     )
   }
 
-  enable_flow_logs = var.enable_flow_logging
+  log_config {
+    aggregation_interval = "INTERVAL_10_MIN"
+    flow_sampling        = 0.5
+    metadata             = "INCLUDE_ALL_METADATA"
+  }
```

Comment:
Can you actually replace this with a dynamic block, and expose these as a variable so they can be configured? Specifically, there should be three new variables `flow_logs_aggregation_interval`, `flow_logs_sampling`, and `flow_logs_metadata` that only populate this config if `enable_flow_logging` is `true`.